arm/Microsoft.OperationalInsights/workspaces/deploy.bicep

```diff
@@ -54,6 +54,23 @@ param publicNetworkAccessForQuery string = 'Enabled'
 @description('Optional. Set to \'true\' to use resource or workspace permissions and \'false\' (or leave empty) to require workspace permissions.')
```

Comment:
For all: @senavar, any chance you tested the code in pipelines? If so, could you do me a favor and add the pipeline badges to your PR summary for the pipelines to which you added the diagnostic settings?

Reply:
Badges have been added!